test/Readout.spec.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-undef */
+describe('The readout', function() {
+    let page;
+
+    before(async function() {
+        page = await browser.newPage();
+        await page.goto('http://localhost:8080');
+    });
+
+    after(async function() {
+        await page.close();
+    });
+
+    let checkForSingularElement = async function(selector) {
```

Comment:
Since this function turned out to be just two lines of code, I think you should [inline the function](https://refactoring.com/catalog/inlineFunction.html). That will make these tests more consistent with those in `Blurb.spec.js`.